tests/conftest.py

```diff
@@ -228,17 +233,54 @@ def es(app):
     list(current_search.delete(ignore=[404]))
 
 
+def record_indexer_receiver(sender, json=None, record=None, index=None,
+                            **kwargs):
+    """Mock-receiver of a before_record_index signal."""
+    if ES_VERSION[0] == 2:
+        suggest_byyear = {}
+        suggest_byyear['context'] = {
+            'year': json['year']
```

Comment:
Why isn't this one casted using `str` (`'year': [str(json['year'])]`) as the in ES5?

Reply:
ES5 does not accept int in the suggest context